### PR TITLE
feat: add OpenApi spec to feature variants

### PR DESCRIPTION
**Pull request by chriswk**
This adds openapi middleware for the feature variants controller. :)

### CODE REVIEW

File: src/lib/routes/admin-api/project/variants.ts [hunk 1]
```diff
@@ -29,19 +32,61 @@ export default class VariantsController extends Controller {
...
+                    tags: ['admin'],
+                    operationId: 'getFeatureVariants',
+                    responses: { 200: featureVariantsResponse },
+                }),
+            ],
+        });
+        this.route({
+            method: 'patch',
+            path: PREFIX,
+            acceptAnyContentType: true,
```
Comment (olav) [inline src/lib/routes/admin-api/project/variants.ts:58]: Removable.

File: src/lib/routes/admin-api/project/variants.ts [hunk 2]
```diff
@@ -29,19 +32,61 @@ export default class VariantsController extends Controller {
...
+                    operationId: 'patchFeatureVariants',
+                    requestBody: patchRequest,
+                    responses: { 200: featureVariantsResponse },
+                }),
+            ],
+        });
+        this.route({
+            method: 'put',
+            path: PREFIX,
+            acceptAnyContentType: true,
```
Comment (olav) [inline src/lib/routes/admin-api/project/variants.ts:73]: Removable.

**Review by olav — CHANGES_REQUESTED** [review]

File: src/lib/routes/admin-api/project/variants.ts (hunk 2 above)
Comment (chriswk) [reply src/lib/routes/admin-api/project/variants.ts]: e5c7e0a2

File: src/lib/routes/admin-api/project/variants.ts (hunk 1 above)
Comment (chriswk) [reply src/lib/routes/admin-api/project/variants.ts]: e5c7e0a2

File: src/lib/routes/admin-api/project/variants.ts [hunk 3]
```diff
@@ -41,7 +41,6 @@ export default class VariantsController extends Controller {
         this.route({
             method: 'get',
             path: PREFIX,
-            acceptAnyContentType: true,
```
Comment (olav) [inline src/lib/routes/admin-api/project/variants.ts]: Still need `acceptAnyContentType` for `get`, should really improve on that.

**Review by olav — CHANGES_REQUESTED** [review]

File: src/lib/routes/admin-api/project/variants.ts (hunk 3 above)
Comment (chriswk) [reply src/lib/routes/admin-api/project/variants.ts]: Oh, ok. thanks

**Review by olav — APPROVED** [review]

**Pull request merged**